network-api/networkapi/buyersguide/templates/product_page.html

```diff
@@ -52,30 +52,37 @@ <h2 class="h1-heading col-md-10">{{product.name}}</h2>
+                {% trans "Five basic steps every company should take to protect consumer privacy." %}
+                <a href="/privacynotincluded/about/meets-minimum-security-standards">{% trans "Learn more." %}</a>
               </p>
             </div>
           </div>
 
           <div class="details mt-2">
             <div class="criterion rating">
-              {% include "fragments/product_criterion_primary_info.html"  with  label="Overall Security Rating" %}
+              {% trans "Overall Security Rating" as overall_security_rating %}
+              {% include "fragments/product_criterion_primary_info.html"  with label=overall_security_rating %}
               <div>
-                <span class="security-score mb-0">{{security_score}}</span><span class="full-security-score mb-0">/{{full_security_score}}</span>
-                <img src="/_images/buyers-guide/score/icon-star.svg" width="20" height="20" alt="star">
+                <span class="security-score mb-0">{{security_score|localize}}</span><span class="full-security-score mb-0">/{{full_security_score}}</span>
+                <img src="/_images/buyers-guide/score/icon-star.svg" width="20" height="20" alt="{% trans "star" context "Used as a noun. Accessibility label for star image" %}">
               </div>
             </div>
-            {% include "fragments/product_criterion.html"  with  show_value_as_symbol=True  value=product.uses_encryption         help=product.uses_encryption_helptext         label="Encryption"              %}
-            {% include "fragments/product_criterion.html"  with  show_value_as_symbol=True  value=product.security_updates        help=product.security_updates_helptext        label="Security updates"        %}
-            {% include "fragments/product_criterion.html"  with  show_value_as_symbol=True  value=product.strong_password         help=product.strong_password_helptext         label="Strong password"         %}
-            {% include "fragments/product_criterion.html"  with  show_value_as_symbol=True  value=product.manage_vulnerabilities  help=product.manage_vulnerabilities_helptext  label="Manages vulnerabilities" %}
-            {% include "fragments/product_criterion.html"  with  show_value_as_symbol=True  value=product.privacy_policy          help=product.privacy_policy_helptext          label="Privacy policy"          %}
+            {% trans "Encryption"              as encryption %}
+            {% trans "Security updates"        as security_updates %}
+            {% trans "Strong password"         as strong_password %}
+            {% trans "Manages vulnerabilities" as manages_vulnerabilities %}
+            {% trans "Privacy policy"          as privacy_policy %}
+
+            {% include "fragments/product_criterion.html"  with  show_value_as_symbol=True  value=product.uses_encryption         help=product.uses_encryption_helptext         label=encryption                %}
```

Comment:
it looks like we've lost the checkmark/X icons in the summary view somewhere in this change; they're now showing as the text "Yes" and "No"

Reply:
![image](https://user-images.githubusercontent.com/177243/77092942-8c365200-69c7-11ea-8a96-0ea9da11e839.png)